### PR TITLE
i18n : intllib po/pot method, added french translation

### PR DESCRIPTION
**Pull request by fat115**
plural forms can be improved (todo)

---
traduction quasi complète, les formes plurielles sont à améliorer.
à vous de voir ;)

### CODE REVIEW

**Comment by ObaniGemini** [comment]
Not a bad idea, but was it really needed to duplicate each return line ?

**Comment by ObaniGemini** [comment]
MFF is known to be a mass of bloat code, should we add to the mass ? @MinetestForFun/devs

**Comment by fat115** [comment]
What do you mean exactly by "duplicate each return line" ?

**Comment by ObaniGemini** [comment]
@fat115 : don't mind what I said, I totally misread the GIT diff (I suppose I'm sleepy...).
I'm not against it, I've seen some players complaining about some things not being translated in-game, so why not ?

**Comment by Lymkwi** [comment]
I'm not sure that making areas conditionally load intllib is worth it, especially since MinetestForFun's modpack is already bloated with tons of code, and most people over on our servers understand english, so I doubt translations are that useful. The UI is also intuitive enough to understand after a short while.
I could, however, merge your changes into a separate branch, for an alternate version of areas with translation support, in case someone out there is interested in getting i18n translations.

File: init.lua [hunk 1]
```diff
@@ -20,20 +25,19 @@ dofile(areas.modpath.."/hud.lua")
...
 })
 
 if not minetest.registered_privileges[areas.config.self_protection_privilege] then
 	minetest.register_privilege(areas.config.self_protection_privilege, {
-		description = "Can protect areas.",
+		description = S("Can protect areas."),
 	})
 end
 
 if minetest.setting_getbool("log_mod") then
```
Comment (Panquesito7) [inline init.lua]: Use `log_mods` instead.
```suggestion
if minetest.settings:get_bool("log_mods") then
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Looks good. 👍

**Comment by BetterToAutomateTheWorld** [comment]
@Panquesito7 Seems good to me, you can merge it :)

**Pull request merged**